yarn.lock

```diff
@@ -6912,11 +6912,31 @@
     "@types/node" "*"
     form-data "^3.0.0"
 
-"@types/node@*", "@types/node@12.12.50", "@types/node@>=12.12.47", "@types/node@>=13.7.0", "@types/node@^10.17.55", "@types/node@^10.17.60", "@types/node@^12", "@types/node@^12.12.17", "@types/node@^14.14.35":
+"@types/node@*", "@types/node@>=12.12.47", "@types/node@^12", "@types/node@^12.12.17":
```

Comment:
Something is wrong with this lock file.

<img width="309" alt="image" src="https://user-images.githubusercontent.com/572096/221584268-d95e4161-f995-4c89-961f-509fcc0b4103.png">